rest-service/manager_rest/test/endpoints/test_blueprints.py

```diff
@@ -36,18 +37,18 @@ def test_get_empty(self):
     def test_get_nonexistent_blueprint(self):
         try:
             self.client.blueprints.get('15')
-        except CloudifyClientError, e:
+        except exceptions.CloudifyClientError, e:
```

Comment:
if we're already changing those lines, please also change the `except E, e` to `except E as e`, which because the comma syntax is old and deprecated (and stops working in py3)